### DISCUSSION TITLE
why dot not integrate  joomcck with joomvideo and others solutions?

### DISCUSSION THREAD

**Discussion by CMS-CORP** [JoomCoder-com/JoomCCK#252] [Ideas]
many of my professional customers can't find what they're looking for in wordpress when it comes to uploading video files to the front end (via custom form to post)  . even the powerful ACF handles this very poorly. the power of joomcck with the power of joomvideo would be incredible and could attract professional customers.

**Comment by andiamine** [comment]
Hi @CMS-CORP i think with JoomCCK only you can achieve without integration with JoomVideos.

**Reply by devforcrypto** [reply]
would  be good to have a youtube like template for joomcck, with playlist feature, ect like additional pack to buy
